third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3567,6 +3601,18 @@ func flattenMaintenancePolicy(mp *containerBeta.MaintenancePolicy) []map[string]
 	if mp == nil || mp.Window == nil {
 		return nil
 	}
+
+	exclusionWindows := []map[string]interface{}{}
```

Comment:
`exclusions` or `maintenanceExclusions` rather than use `window` which is not used in the names of these objects?

Reply:
changed to exclusions